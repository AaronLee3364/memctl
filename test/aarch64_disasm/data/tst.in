fffffff0070c0000 ea0600df
