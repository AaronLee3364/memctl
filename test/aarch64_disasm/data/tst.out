fffffff0070c0000  TST     X6, X6
